src/Symfony/Component/Cache/CacheItem.php

```diff
@@ -151,7 +151,7 @@ public function getMetadata(): array
     /**
      * Validates a cache key according to PSR-6.
      *
-     * @param string $key The key to validate
+     * @param mixed $key The key to validate
```

Comment:
Isn't better to keep `string` since it's the only working type ?

Reply:
Actually string was wrong: this method validates any type